app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, file_name, move_to = command.split()
+    directories = []
+
+    if "/" in move_to:
+        move_to = move_to.split("/")
```

Comment:
Is ur code cross-platform ?) For example, will it work on win with \ ? Use os.path.split() instead